go/tao/tpm2_tao.go

```diff
@@ -285,47 +291,100 @@ func NewTPM2Tao(tpmPath string, statePath string, pcrNums []int) (Tao, error) {
 	tt.pcrs = pcrNums
 	tt.path = statePath
 
-	// Create the root key.
 	keySize := uint16(2048)
 	quotePassword := ""
 
-	rootSaveContext := path.Join(tt.path, "root_context")
-	_, rootErr := os.Stat(rootSaveContext)
+	quoteKeyPrivateBlobFile := path.Join(tt.path, "quote_private_key_blob")
+	_, quotePrivateErr := os.Stat(quoteKeyPrivateBlobFile)
+	quoteKeyPublicBlobFile := path.Join(tt.path, "quote_public_key_blob")
+	_, quotePublicErr := os.Stat(quoteKeyPrivateBlobFile)
```

Comment:
I think this is supposed to be
_, quotePublicErr := os.Stat(quoteKeyPublicBlobFile)
and same for line 305.